.storybook/main.js

```diff
@@ -0,0 +1,51 @@
+import { dirname, join } from "path";
+/*
+Copyright 2022 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License.
+*/
+const config = {
+    stories: [
+        '../packages/*/stories/*.stories.js',
+        '../tools/*/stories/*.stories.js',
+    ],
+
+    addons: [
+        getAbsolutePath("@storybook/addon-links"),
```

Comment:
```suggestion
        "@storybook/addon-links",
```

We should be able to reference the package by string name rather than the full path. This prevents us having to maintain a hardcoded pathing infrastructure.

Reply:
Yeah, I have no idea why `getAbsolutePath()` was being used here, but I went ahead and removed it.